docs/deploy/create_server.rst

```diff
@@ -115,49 +115,60 @@ If you couldn't select Ubuntu above, follow these additional steps:
 
       reboot
 
-2. Apply core changes
+2. Set up DNS records
+---------------------
+
+#. Create a new hostname DNS entry in `GoDaddy <https://dcc.godaddy.com/manage/OPEN-CONTRACTING.ORG/dns>`__
+
+All server hostnames should be in the following format ocpXX.open-contracting.org (e.g. ocp01, ocp02 etc).
+Each server should have a unique hostname so for each new server, increment the counter by one. 
+To see which hostnames are currently in use `refer to the salt roster config <https://github.com/open-contracting/deploy/blob/master/salt-config/roster>__`.
+
+
+3. Apply core changes
 ---------------------
 
 #. Connect to the server as the ``root`` user using SSH, and change its password, using the ``passwd`` command. Use a `strong password <https://www.lastpass.com/password-generator>`__, and save it to OCP's `LastPass <https://www.lastpass.com>`__ account.
 
-   .. note::
+.. note::
 
-      The root password is needed if you can't login via SSH (for example, due to a broken configuration). For Bytemark, open the `panel <https://panel.bytemark.co.uk/servers>`__, click the server's *Console* button, and login.
+   The root password is needed if you can't login via SSH (for example, due to a broken configuration). For Bytemark, open the `panel <https://panel.bytemark.co.uk/servers>`__, click the server's *Console* button, and login.
 
-#. Add a target to the ``salt-config/roster`` file in this repository, using the hostname from above. If the service is an instance of `CoVE <https://github.com/OpenDataServices/cove>`__, choose a target name starting with ``cove-``.
+#. Add a target to the ``salt-config/roster`` file in this repository, using a logical target name matching the service. If the service is an instance of `CoVE <https://github.com/OpenDataServices/cove>`__, choose a target name starting with ``cove-``.
 
-#. `Upgrade packages <https://docs.saltstack.com/en/latest/ref/modules/all/salt.modules.aptpkg.html#salt.modules.aptpkg.upgrade>`__:
+#. `Run the onboarding state file <https://github.com/open-contracting/deploy/blob/master/salt/onboarding.sls>__`
 
-   .. code-block:: bash
+This state file ensures that the system is patched, configures the system hostname and applies the core salt configs.
 
-      salt-ssh TARGET pkg.upgrade dist_upgrade=True
+Replace "ocpXX" with the hostname you set up in GoDaddy earlier.
```

Comment:
Put these on the same line, otherwise I think the numbered list item won't render correctly.

Reply:
Tweaked + numbering looks good in GitHub.